src/schemata/definitions/workflow_template/action_objects/updateResource.yml

```diff
@@ -33,12 +33,55 @@ additionalProperties:
               type: object
               additionalProperties: false
               properties:
+                custom_attributes:
+                  $ref: '#/definitions/script'
+                  description: |
+                    To update a custom attribute, the script must resolve to a hash object with `name`, `type` and `value` attributes. The supported custom attribute data types are `number`, `text`, `boolean` (checkbox), `date`, `datetime`, `file`, `image`, `physicalQuantity`, `relation`, and `duration`.
                 title:
                   $ref: '#/definitions/script'
-
+        - type: object
+          required:
+            - type
+            - attributes
+          properties:
+            type:
+              type: string
+              enum:
+                - collection
+                - connector
+                - dashboard
+                - device
+                - export
+                - measurement
+                - sample
+                - workflow_step_template 
+                - workflow_template
+            attributes:
+              type: object
+              additionalProperties: false
+              properties:
+                custom_attributes:
+                  $ref: '#/definitions/script'
+                  description: |
+                    To update a custom attribute, the script must evaluate to a hash object with `name`, `type` and `value` attributes. The supported custom attribute data types are `number`, `text`, `boolean` (checkbox), `date`, `datetime`, `file`, `image`, `physicalQuantity`, `relation`, and `duration`.
 examples:
-  - update_resource:
-      id: workflow_run.data.id
-      type: workflow_run
-      attributes:
-        title: sampleField.name
+  - id: workflow_run.data.id
```

Comment:
why did you remove the `update_resource:` key from the examples?

Reply:
I added the key